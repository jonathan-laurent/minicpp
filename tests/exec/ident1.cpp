#include <iostream>

int main() {
  int ___________ = 1;
  int ____1_____023__ = 0;
  if (___________ && ____1_____023__ == 0)
    std::cout << "non, tous les programmes n'affichent pas hello world\n";
}
