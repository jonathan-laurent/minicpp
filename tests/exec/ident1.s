	.text
	jal  _p0__main
	li   $v0, 10
	syscall
_p0__main:
	addi $sp, $sp, -4
	li   $t0, 1
	li   $t1, 0
	bnez $t0, L16
L10:
	addi $sp, $sp, 4
	jr   $ra
L16:
	li   $t0, 0
	bne  $t1, $t0, L10
	la   $a0, _s__0
	li   $v0, 4
	syscall
	b    L10
	b    L10
	.data
_s__0:
	.asciiz "non, tous les programmes n'affichent pas hello world\n"

