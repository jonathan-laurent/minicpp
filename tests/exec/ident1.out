non, tous les programmes n'affichent pas hello world
